### DISCUSSION TITLE
Why not Jekyll?

### DISCUSSION THREAD

**Discussion by xplosionmind** [bridgetownrb/bridgetown#432] [General]
Hello everybody. I am a proud and overall quite happy Jekyll user.

What are the arguments which should convince me to switch to Bridgetown?  
I read the most relevant articles on the Bridgetown blog and I get its potential and power, while I understand that Jekyll is not likely to be further developed in the future; yet, strictly feature-wise, I cannot grasp the core changes which would make the user experience so much better.

I cannot find anywhere a proper <q>Jekyll vs Bridgetown</q> review/comparison; with the right answers and references in this discussion, I may write one myself.

Thank you very much!

**Comment by xplosionmind** [comment]
For what concerns me in particular, I am not a developer and I am worried that my very personal and sophisticated Jekyll website structure and workflow, years in the making, would take a long time and great efforts to be converted and compatible with Bridgetown. How can I be sure that Bridgetown is going to be a stable and viable long-term solution?

Concerning plugins, below are the ones I am using, and they are fundamental to me. Without a way to make them work on Bridgetown, I am forced to stick with Jekyll.

- jekyll-feed (I know there is bridgetown-feed)
- jekyll-seo-tag (I know there is bridgetown-seo-tag)
- jekyll-sitemap
- jekyll-last-modified-at
- jekyll-target-blank
- jekyll-watch (I know it is strictly Jekyll-related, hence useless in Bridgetown)
- jekyll-redirect-from
- jekyll-debug (I know it is strictly Jekyll-related, hence useless in Bridgetown)
- jekyll-mentions
- jekyll-auto-image
- jekyll-email-protect
- jekyll-paginate-v2
- jekyll-wikilinks
- jekyll-commonmark
- jekyll-graph

**Comment by jaredcwhite** [comment]
@xplosionmind Thanks for your question! You may be surprised by my answer, but I think you'd be well served by just sticking with Jekyll! There's much wisdom in the adage "if it ain't broke, don't fix it". While I would argue Bridgetown offers a lot of "new" features, unless you urgently need/want to take advantage of them, switching to Bridgetown would be a lot of effort for little immediate gain.

In terms of plugins, many Jekyll plugins can be ported over (and a few have, including sitemap), but again, a fair degree of effort to do so. FYI, the features in jekyll-paginate-v2 are built-in to Bridgetown so that's helpful.

At some point when you want to tackle more of a redesign/revamp of your site, or build a whole new site of some kind, that'd be a logical point to use Bridgetown going forward.

**Comment by brandonzylstra** [comment]
I don't disagree with @jaredcwhite at all, but I would add one detail: since something made you curious about Bridgetown in the first place, you might find that just spending some spare time on a Saturday afternoon trying out Bridgetown (on a new site at first, to get familiar with it) might help satisfy that curiosity and also give you a lot more insight into whether Bridgetown is something you'd ever want to convert your Jekyll site to.

You might find that you love it.  You might find that none of what it offers is useful to you.  Or somewhere in between.

**Reply by kigster** [reply]
I found that keeping my Jekyll sites working as the rest of Ruby and the gems are updated is getting harder and harder. Some Plug-ins no longer work with Ruby 3.2, and maintainers have abandoned the projects. This means sooner or later it will be next to impossible to maintain a Jekyll site unless you are not using any plugins.

I just discovered Bridgetown and I can't wait to migrate my https://kig.re/ blog over. Not so much due to the new features but because Jekyll seems to have lost momentum and sooner or later it will just stop working.
